plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/cache/CachingHiveMetastore.java

```diff
@@ -1111,24 +1126,14 @@ public <K, V> LoadingCache<K, V> buildCache(com.google.common.base.Function<K, V
             }
 
             @Override
-            public <K, V> LoadingCache<K, V> buildBulkCache(Function<Iterable<K>, Map<K, V>> bulkLoader)
+            public <K, V> Cache<K, AtomicReference<V>> buildBulkCache()
             {
                 // disable refresh since it can't use the bulk loading and causes too many requests
-                return CachingHiveMetastore.buildCache(expiresAfterWriteMillis, maximumSize, statsRecording, bulkLoader);
+                return CachingHiveMetastore.buildBulkCache(expiresAfterWriteMillis, maximumSize, statsRecording);
             }
         };
     }
 
-    private static CacheFactory neverCacheFactory()
-    {
-        return cacheFactory(
-                OptionalLong.of(0),
```

Comment:
Would it be possible for the second commit not to include moving `neverCacheFactory()` around?